src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -49,8 +51,8 @@
     DifferentialDrive m_drive = new DifferentialDrive(m_left, m_right);
 
     //defines values that will be used in the PIDController (In order of where they will fall in the Controller)
-    final static double kP = 1.0 / 100;
-    final static double kI = 0.000085;
+    final static double kP = 0.014;
```

Comment:
I would make these preferences like we did in the Lift and Manipulator (we seemed to have found the proper way to get them showing up on Shuffleboard for easy editing)

Reply:
We got this working pretty well yesterday just using the PID controller object on smartdashboard. 
If we want to use these as preferences we have to add a method to reload the settings, which we currently don't do.